docs/references/configuration.rst

```diff
@@ -599,11 +599,13 @@ jwt-aud
   **In-Database** pgrst.jwt_aud
   =============== =================================
 
-  Specifies the `JWT audience claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3>`_. If this claim is present in the client provided JWT then you must set this to the same value as in the JWT, otherwise verifying the JWT will fail.
+  Specifies the `JWT audience claim <https://datatracker.ietf.org/doc/html/rfc7519#section-4.1.3>`_.
+  If this claim is present in the client provided JWT then you must set this to the same value as in the JWT, otherwise verifying the JWT will fail.
```

Comment:
>   If this claim is present in the client provided JWT then you must set this to the same value as in the JWT

Can you clarify this? Don't quite get it.

Reply:
Yeah, I attempted an improvement there. Thanks to this I noticed that I was forgetting to add what we mentioned here: https://github.com/PostgREST/postgrest/pull/4043#discussion_r2061257712.